schema.graphql

```diff
@@ -33,3 +32,82 @@ type AuctionedName @entity {
   bidCount: Int!
   state: AuctionState
 }
+
+type Resolver @entity {
+  id: ID!
+  domain: Bytes!
```

Comment:
Shouldn't this be a link back to the Domain entity?

Reply:
Yep, this was missed, will add